examples/llama.py

```diff
@@ -190,6 +190,11 @@ def load(fn:str):
   else:
     return torch_load(fn)
 
+def fetch_tiny_llama():
+  print("No model specified. Fetching tiny LLaMA...")
```

Comment:
does not make sense to have this print inside the function

Reply:
Ok.